modules/secret_manager/versions.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_version = ">= 0.13.0"
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 4.35"
+    }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = "~> 4.35"
+    }
+     kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "~> 2.10"
+    }
+ }
+}
```

Comment:
can you add a small README.md on this module explaining why and how to use it ?

Reply:
On its way 👍